src/lib/api.js

```diff
@@ -85,11 +85,11 @@ const extractSortby = function (params) {
       const sortbys = sortby.split(',')
       sortbys.forEach((sortbyRule) => {
         if (sortbyRule[0] === '-') {
-          sortbyRules.push({ 'field': sortbyRule.slice(1), 'direction': 'desc' })
+          sortbyRules.push({ field: sortbyRule.slice(1), direction: 'desc' })
```

Comment:
This should be using `map` rather than `forEach` with a bunch of pushes.

Reply:
filed #203